source/support.tex

```diff
@@ -3166,6 +3168,214 @@
 An \impldef{return value of \tcode{bad_typeid::what}} \ntbs{}.
 \end{itemdescr}
 
+\rSec1[support.srcloc]{Class \tcode{source_location}}
+
+The header \tcode{<source_location>} defines the class \tcode{source_location} that provides a means to obtain source location information.
+
+\rSec2[source_location.syn]{Header \tcode{<source_location>} synopsis}
+
+\begin{codeblock}
+namespace std {
+  struct source_location {
+    // source location construction
+    static consteval source_location current() noexcept;
+    constexpr source_location() noexcept;
+
+    // source location field access
+    constexpr uint_least32_t line() const noexcept;
+    constexpr uint_least32_t column() const noexcept;
+    constexpr const char* file_name() const noexcept;
+    constexpr const char* function_name() const noexcept;
+
+  private:
+    uint_least32_t line_;               // \expos
+    uint_least32_t column_;             // \expos
+    const char* file_name_;             // \expos
+    const char* function_name_;         // \expos
+  };
+}
+\end{codeblock}
+
+\pnum
+The type \tcode{source_location} meets the
+\oldconcept{DefaultConstructible},
+\oldconcept{CopyConstructible},
+\oldconcept{Copy\-Assignable}, and
+\oldconcept{Destructible}
+requirements\iref{utility.arg.requirements}.
```

Comment:
The paper wanted a new paragraph here.

Reply:
Yes, but single-sentence paragraphs are bad. Both this paragraph and the next describe requirements for the implementation-provided source_location type, so not having a new paragraph seems reasonable.

Reply:
Then combine it with the next paragraph?  otherwise, please obey the paper and place in a separate paragraph - it does not belong with this paragraph.

Reply:
Yes, sounds reasonable. Done.